challenges/pick/prompt.md

```diff
@@ -0,0 +1,149 @@
+## The Problem `Pick` Solves
+
+`Pick` is a very useful helper type that allows you to create a type by using an existing type as a starting point.
+
+Say, for example, your codebase has a type that looks like this:
+
+```ts
+interface Pokemon {
+  name: string;
+  type: string;
+  hitPoints: number;
+  stage: string;
+  evolutionStage: number;
+  attacks: string[];
+  weakness: string;
+  resilience: string;
+}
+```
+
+That's a lot of properties!  But what if you have a function that only needs two properties from this type:
+
+```ts
+const pickYourPokemon = (pokemon: Pokemon) => {
+  const { type, name } = pokemon;
+
+  return `You picked the ${type}-type Pokemon ${name}!
```

Comment:
```suggestion
  return `You picked the ${type}-type Pokemon ${name}!`
```

Reply:
good catch: fixed in 891ee1ba12f9584965daa2b3e0d407b22847e81b